src/main/java/me/zeroeightsix/kami/module/modules/render/NoMaps.kt

```diff
@@ -0,0 +1,27 @@
+package me.zeroeightsix.kami.module.modules.render
+
+import me.zeroeightsix.kami.module.Module
+
+/**
+ * Created by littlebroto1 on 8/29/2020
+ */
+
+@Module.Info(
+        name = "NoMaps",
+        category = Module.Category.RENDER,
+        description = "Replaces maps with a family friendly alternative"
+)
+class NoMaps : Module() {
+    companion object {
```

Comment:
this entire thing isn't needed

Reply:
Opps, I forgot to remove the inside of the module. This was one of my many failed attempts to get this working.

Reply:
Let me guess, Intellij kotlinizing error? :p

Reply:
I was having issues getting whether NoMaps was enabled or not using the module manager.